docs/roles/aws/aws_iam_saml.md

```diff
@@ -69,7 +69,9 @@ This requires boto and AWS-CLI on the provisioning server.
 ```yaml
 aws_iam_saml:
   aws_profile: "{{ _aws_profile }}" # Boto profile to use for AWS connections
-  tags: {} # Dict of AWS tags to apply
+  region: "eu-west-2" # AWS region to use
+  tags: {} # List of AWS tags to apply
```

Comment:
This is wrong, probably accepted the wrong bit in the PR, `tags: {}` is a dict but the comment says `List of AWS tags`.

Reply:
Sorted!